[Pages] _redirects proxying functionality (#7613)
cloudflare/cloudflare-docs: content/pages/platform/redirects.md

Similarly, the matched value can be used in the redirect location with `:placeholder_name`.

#### Proxying

Proxying will only support relative URLs on your site - i.e. you can't proxy external domains.

Only the first redirect will apply, for example, in the following scenario, a request to `/a` will render `/b`, and a request to `/b` will render `/c`, but `/a` will not render `/c`.

```
/a /b 200
/b /c 200
```

{{<Aside heading="Proxying's impact on SEO">}}

> Be aware that proxying pages can have an adverse effect on search engine optimization (SEO). Search engines often penalize websites that serve duplicate content, so consider adding a `Link` HTTP header which informs search engines of the canonical source of content.
> 
> For example, if you have added `/about/faq/* /about/faqs 200` to your `_redirects` file, you will likely also want to add the following to your `_headers` file:

Reviewer: ```suggestion
Be aware that proxying pages can have an adverse effect on search engine optimization (SEO). Search engines often penalize websites that serve duplicate content. Consider adding a `Link` HTTP header which informs search engines of the canonical source of content.

For example, if you have added `/about/faq/* /about/faqs 200` to your `_redirects` file, you may want to add the following to your `_headers` file:
```
